tests/integration/tables/diagnostics.cpp

```diff
@@ -0,0 +1,44 @@
+#include <osquery/tests/integration/tables/helper.h>
+
+namespace osquery {
+namespace table_tests {
+class DiagnosticsTest : public testing::Test {
+ protected:
+  void SetUp() override {
+    setUpEnvironment();
+  }
+};
+
+TEST_F(DiagnosticsTest, test_sanity) {
+  QueryData const rows = execute_query("select * from diagnostics");
+
+  QueryData const specific_query_rows = execute_query(
```

Comment:
I like that you have both of these tests, but I'd rather they were grouped a bit differently. Maybe groups the query, the assert, and the validate together? Easier to catch errors in the ASSERT